.github/pull_request_template.md

```diff
@@ -35,11 +35,9 @@
 * [ ] _All_ future TODOs are captured in issues, which are referenced
       in code comments.
 * [ ] All relevant type-of-change labels have been added.
-* [ ] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md)
-      document.
-* [ ] These code changes follow [cisagov code
-      standards](https://github.com/cisagov/development-guide).
+* [ ] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
+* [ ] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
 * [ ] All relevant repo and/or project documentation has been updated
       to reflect the changes in this PR.
-* [ ] Tests have been added to cover the changes in this PR.
+* [ ] Tests have been added and/or modified to cover the changes in this PR.
```

Comment:
Would this be a simpler catch-all to use?
```suggestion
* [ ] Tests have been updated to cover the changes in this PR.
```

Reply:
I think it's worth calling out "added" to put it in peoples' heads in case that hadn't occurred to them.

Reply:
I like the slightly more explicit language in the original.